plugin/__init__.py

```diff
@@ -468,10 +468,15 @@ def removeDeckConfigId(self, configId):
 
 
     @util.api()
-    def storeMediaFile(self, filename, data=None, url=None):
+    def storeMediaFile(self, filename, data=None, file=None, url=None):
```

Comment:
Looking at this a little bit further, we probably need to have a better name for `file`, as it does not make it clear what it is being used for. I would recommend `dataPath` to make it clear that it is where the data is coming from. The docs will need to also be updated to reflect this change.